backend/app/usecase/validator/longlink.go

```diff
@@ -10,20 +10,16 @@ type LongLink struct {
 }
 
 // IsValid checks whether the given long link has valid format.
-func (l LongLink) IsValid(longLink *string) bool {
-	if longLink == nil {
+func (l LongLink) IsValid(longLink string) bool {
+	if longLink == "" {
```

Comment:
```suggestion
	if len(longLink) == 0 {
```
Do you think checking for length makes it easier to read and is the same as the check below for max length?

Reply:
I'm not sure if it makes it any easier to read, but I do agree that it makes it more consistent with the other length check. 👍